packages/@aws-cdk/aws-ecs/lib/container-image.ts

```diff
@@ -0,0 +1,40 @@
+import { ContainerDefinition } from './container-definition';
+
+/**
+ * Base class for container images
+ */
+export abstract class ContainerImage {
+  /**
+   * Name of the image
+   */
+  public abstract readonly imageName: string;
+
+  /**
+   * Called when the image is used by a ContainerDefinition
+   */
+  public abstract bind(containerDefinition: ContainerDefinition): void;
```

Comment:
What is the value of this method?  It doesn't actually seem to bind to this specific image object; it just sets a flag back in the containerDefinition?